packages/graBing/PKGBUILD

```diff
@@ -0,0 +1,36 @@
+# This file is part of BlackArch Linux ( http://blackarch.org ).
+# See COPYING for license details.
+
+pkgname="graBing"
```

Comment:
I'm not sure about the `B` (capital letter) there. But I would change it to `b`.
Maybe, I hope, someone else more experienced than me will say.

Reply:
If I change this to `b` the pkgbuild it will not work due the source package name, otherwise I need to change it.